sql/core/src/main/scala/org/apache/spark/sql/execution/window/WindowExecBase.scala

```diff
@@ -149,8 +150,8 @@ trait WindowExecBase extends UnaryExecNode {
             case AggregateExpression(f, _, _, _, _) => collect("AGGREGATE", frame, e, f)
             case f: FrameLessOffsetWindowFunction =>
               collect("FRAME_LESS_OFFSET", f.fakeFrame, e, f)
-            case f: OffsetWindowFunction if !f.ignoreNulls &&
-              frame.frameType == RowFrame && frame.lower == UnboundedPreceding =>
+            case f: OffsetWindowFunction if frame.frameType == RowFrame &&
+              frame.lower == UnboundedPreceding =>
               frame.upper match {
```

Comment:
if we go with the `case _ => collect("AGGREGATE", frame, e, f)` branch, do we still support ignore nulls?

Reply:
Nth_value support ignore nulls. lead and lag not.